qemu/tests/cfg/block_boot_multi_disks.cfg

```diff
@@ -0,0 +1,48 @@
+- block_boot_multi_disks:
+    type = block_boot_multi_disks
+    start_vm = no
+    no s390 s390x ppc64 ppc64le
+    vt_ulimit_nofile = 65536
+    force_create_image = yes
+    force_create_image_image1 = no
+    remove_image = yes
+    remove_image_image1 = no
+
+    stg_image_size = 512M
+    stg_image_format = qcow2
+    stg_image_name = "images/%s"
+    stg_drive_format = virtio
+
+    check_disk_num = 6
+    login_timout = 360
+
+    Linux:
+        tmp_dir = /var/tmp/test
+        check_num_cmd = "lsblk -nd|grep [sv]d|wc -l"
+        guest_cmd = "mkdir -p ${tmp_dir} && mkfs.xfs -f {0}  &&"
+        guest_cmd += " mount -t xfs {0} ${tmp_dir} && dd if=/dev/zero "
+        guest_cmd += " of=${tmp_dir}/test.img bs=1M count=256 oflag=direct && "
+        guest_cmd += " umount ${tmp_dir}"
+    Windows:
+        check_num_cmd = powershell -command (get-disk).count
+        guest_cmd = "WIN_UTILS:\coreutils\DummyCMD.exe {} 102400000 1"
+    variants:
+        - with_default_bios:
+            only default_bios
+            check_message = "WARNING - Unable to allocate resource at add_drive"
+            variants:
+                - with_no_warn_check:
+                    expect_find = no
+                    stg_image_num = 15
+                - with_warn_check:
+                    expect_find = yes
+                    stg_image_num = 16
+            aarch64:
+                only with_no_warn_check
+                check_message =
+                stg_image_num = 99
```

Comment:
Why add this configuration separately on aarch64? 
When `stg_image_num = 99` guest fails with timeout and when I delete it, using `stg_image_num = 15` provided above cases pass